docs/migration-config/README.md

```diff
@@ -0,0 +1,34 @@
+# Migration guide
+
+In case you want to quickly migrate from the old stack into the new stack, you can use the following two configurations to help you to migrate without changing too much fields.
+
+You just need to drop the config, use it in the `--config` args and add some additional environment variables.
+
+Please take a deeper look in the config, but 90% of the time you'll be safe just using it. If you find any issues using this config, feel free to open an issue.
```

Comment:
If you check [a Markdown guide](https://google.github.io/styleguide/docguide/style.html#:~:text=Markdown%20content%20follows%20the%20residual,an%2080%2Dcharacter%20line%20limit.) they say .md is supposed be formatted so that if you used `cat` in a terminal and thus limited to 80 chars per line. Up to you.